### PR TITLE
Simple major.minor version checking implemented.

### LINKED ISSUE #20: Add buildfox_required_version variable
Ninja have `ninja_required_version` to control manifest versioning, we need something similar.

### PR DESCRIPTION
**Pull request by den-mentiei**
Fixes #20.

### CODE REVIEW

**Comment by den-mentiei** [comment]
We need to think on http://semver.org support maybe. But I wanted to keep things simple (as they are in ninja).

File: lib_version.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import re
+
+MAJOR = 1
+MINOR = 0
+VERSION = "%d.%d" % (MAJOR, MINOR)
+
+# Simple major.minor matcher
+re_version = re.compile(r"^(\d+)\.(\d+)$")
+
+def parse(version):
```
Comment (jimon) [inline lib_version.py:12]: hey, no need to complicate stuff, here is one from [here](http://stackoverflow.com/questions/11887762/how-to-compare-version-style-strings) :

```
def versiontuple(v):
    return tuple(map(int, (v.split("."))))

versiontuple("2.3.1") > versiontuple("10.1.1")
```

And this function can be put in lib_util.py

File: buildfox.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# BuildFox ninja generator
+#!/usr/bin/python3
```
Comment (jimon) [inline buildfox.py:1]: Maybe just put `# BuildFox ninja generator` after.

Comment (den-mentiei) [inline buildfox.py:1]: Yep, missed it.

File: lib_version.py (hunk 1 above)
Comment (den-mentiei) [inline lib_version.py:12]: I think we should "complicate" stuff as it's connected with user input/output and good error handling is a must.

Also, regex solution is not much longer than `versiontuple`, but it's more fail-proof :)

**Pull request merged**